src/styles/nav.scss

```diff
@@ -0,0 +1,49 @@
+.nav {
+  display: flex;
+
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style: none;
+    text-transform: uppercase;
+  }
+
+  &__list-el {
+    display: flex;
+
+    &:not(:first-child) {
+      margin-left: 20px;
+    }
+  }
+
+  &__list-link {
+    line-height: 60px;
+    text-decoration: none;
+    color: #060b35;
+    position: relative;
+    font-weight: 500;
+
+    &:hover {
+      color: #00acdc;
+    }
+  }
+}
+
+  
+.is-active {
+  color: #00acdc;
+
+  &::after {
+    content: "";
+    position: absolute;
+    width: 37px;
+    height: 4px;
+    top: 56px;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: #00acdc;
```

Comment:
Move all colors to the variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)